tests/integrationv2/utils.py

```diff
@@ -2,6 +2,10 @@
 from providers import S2N, OpenSSL
 
 
+def to_bytes(str):
+    return bytes(str.encode('utf-8'))
```

Comment:
s/`to_bytes`/`str_to_bytes`?

Reply:
I've moved this to the separate refactor PR, but note there that it doesn't actually require a string: https://github.com/aws/s2n-tls/blob/640ed6a78f7a128e606a2b409bd49b289e69b4bc/tests/integrationv2/utils.py#L4-L7